src/index.html

```diff
@@ -7,8 +7,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav__container">
+        <div class="logo__container">
+          <a href="#home">
+            <img src="/images/logo.png" alt="MOYO">
+          </a>
+        </div>
+        <ul class="nav__list">
+          <li><a href="#apple" class="nav_item is-active">Apple</a></li>
```

Comment:
- nope, we don't do it like that. I understand the temptation to provide one-line solution, but our top priority - to have our code maximally readable and maintainable

- it does not seem logical that the link has `nav_item` class, but not the list item (`<li>`)
```suggestion
          <li class="nav__item is-active">
            <a href="#apple" class="nav_item is-active">Apple</a>
          </li>
```